code/game/objects/items/tools/surgery_tools.dm

```diff
@@ -140,13 +147,15 @@
 /obj/item/tool/surgery/bonegel
 	name = "bone gel"
 	icon_state = "bone-gel"
+	OOC_info = "Use to repair bones. close a ribcage or skull by 1. retractor then 2. bone gel, fix a bone by 1. bone gel 2. bone setter. A broken bone cannot be fixed if that body part has more than ~30 brute damage. Give your patient Bicaridine."
```

Comment:
```suggestion
	OOC_info = "Use to repair bones on an open incision through 1. Bone gel 2. Bone Setter. A broken bone cannot be fixed if that body part has more than ~30 brute damage. Give your patient Bicaridine."
```

Reply:
Have to include the skull and ribcage closing bits, else we'll have some unhappy campers.